test/static/turtle_movie.json

```diff
@@ -54,4 +65,3 @@
         }
     }
 }
-
```

Comment:
Text files should end in a newline

See: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline